apps/full-stack-tests/src/components/properties/PresentationFilterBuilderValueRenderer.test.tsx

```diff
@@ -0,0 +1,262 @@
+        builder,
+        modelId: physicalModel.id,
+        categoryId: category.id,
+        classFullName: schema.items.MyPhysicalObjectParent.fullName,
+        userLabel: "Parent Test Class",
+        ["PropertyName"]: "Parent",
+      });
+      const element1 = insertPhysicalElement({
+        builder,
+        modelId: physicalModel.id,
+        categoryId: category.id,
+        classFullName: schema.items.MyPhysicalObject1.fullName,
+        parentId: parentElement.id,
+        userLabel: "Test Class",
+        ["PropertyName"]: "Value1",
+      });
+      const element2 = insertPhysicalElement({
+        builder,
+        modelId: physicalModel.id,
+        categoryId: category.id,
+        classFullName: schema.items.MyPhysicalObject2.fullName,
+        parentId: parentElement.id,
+        userLabel: "Test Class 2",
+        ["PropertyName"]: "Value2",
+      });
+      return { parentElement, element1, element2, category };
+    });
+
+    const testProperty = {
+      name: `#pc_${schemaAlias}_MyPhysicalObjectParent_PropertyName`,
+      displayLabel: "PropertyName",
+      typename: "string",
+    };
+
+    const keys = new KeySet([
+      { id: imodel.element1.id, className: imodel.element1.className },
+      { id: imodel.element2.id, className: imodel.element2.className },
+    ]);
+
+    const testDescriptor = await Presentation.presentation.getContentDescriptor({
+      imodel: imodel.imodel,
+      rulesetOrId: {
+        id: `Test descriptor ruleset`,
+        rules: [
+          {
+            ruleType: "Content",
+            specifications: [
+              {
+                specType: "SelectedNodeInstances",
+              },
+            ],
+          },
+        ],
+      },
+      displayType: DefaultContentDisplayTypes.PropertyPane,
+      keys,
+    });
+
+    if (testDescriptor === undefined) {
+      expect(false);
+      return;
+    }
+
+    const selectedClasses: ClassInfo[] = [
+      {
+        id: imodel.element1.id,
+        name: imodel.element1.className.replace(".", ":"),
+        label: "Test Class",
+      },
+    ];
+
+    const { baseElement, getByRole } = render(
+      <PresentationFilterBuilderValueRenderer
+        property={testProperty}
+        onChange={() => {}}
+        imodel={imodel.imodel}
+        descriptor={testDescriptor}
+        descriptorInputKeys={keys}
+        selectedClasses={selectedClasses}
+        operator={"is-equal"}
+      />,
+    );
+
+    // trigger loadTargets function
+    const user = userEvent.setup();
+    const combobox = await waitFor(() => getByRole("combobox"));
+
+    await waitFor(async () => {
+      await user.click(combobox);
+      expect(queryByText(baseElement, "Value1")).to.not.be.null;
+      expect(queryByText(baseElement, "Value2")).to.be.null;
+    });
+  });
+
+  it("renders 'PresentationFilterBuilderValueRenderer' with correct property values when selected classes are provided without keys", async function () {
```

Comment:
It looks like the descriptorKeys property removal from the test was not saved and the test was indeed a duplicate. For the descriptor it does not matter what ruleset it contains, because we only need to find the field from it, which rule is used to fill the value dropdown depends on whether `PresentationFilterBuilderValueRenderer` has descriptorKeys passed to it. Removed the keys property can rewrite the descriptor rule if that would make the test more readable in your opinion.